source/Databricks/source/SqlStatementExecution/Formats/JsonArrayFormat.cs

```diff
@@ -30,11 +32,16 @@ public JsonArrayFormat(DatabricksSqlStatementOptions options)
     }
 
     public DatabricksStatementRequest GetStatementRequest(DatabricksStatement statement)
-        => new(_options.TimeoutInSeconds, _options.WarehouseId, statement, DatabricksStatementRequest.JsonFormat);
+        => new(_options.WarehouseId, statement, DatabricksStatementRequest.JsonFormat);
 
-    public async IAsyncEnumerable<dynamic> ExecuteAsync(Stream content, DatabricksStatementResponse response)
+    public async IAsyncEnumerable<dynamic> ExecuteAsync(
+        Stream content,
+        DatabricksStatementResponse response,
+        [EnumeratorCancellation]CancellationToken cancellationToken)
     {
-        await foreach (var record in JsonSerializer.DeserializeAsyncEnumerable<string[]>(content))
+        await foreach (var record in JsonSerializer.DeserializeAsyncEnumerable<string[]>(
+                           content,
+                           cancellationToken: cancellationToken))
```

Comment:
Suggestion: In the given scenario it doesn't add value to show the parameter name
```suggestion
                           cancellationToken))
```

Reply:
But it can't be left out either

![image](https://github.com/Energinet-DataHub/geh-core/assets/11583438/bf2c2db4-50e8-4108-acd8-2a9d3159ed73)